sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -92,13 +93,36 @@ class and wrapper class that allows lambda functions to be used as
     'label_from_callable',
     ]
 
+
 T = TypeVar('T')
 PTransformT = TypeVar('PTransformT', bound='PTransform')
 ConstructorFn = Callable[
     [Optional[Any], 'PipelineContext'],
     Any]
 
 
+KNOWN_COMPOSITES = frozenset([
```

Comment:
These are specific to a runner, shouldn't be here.

Reply:
Moved back to original location.